src/GovUk.Education.ExploreEducationStatistics.Data.Model.Tests/Repository/FootnoteRepositoryTests.cs

```diff
@@ -13,247 +15,105 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Tests.Repository
-                                }
-                            }
-                        }
-                    },
-                    new()
-                    {
-                        Footnote = new Footnote
-                        {
-                            Content = "Applies to subject 2",
-                            Order = 6,
-                            Subjects = new List<SubjectFootnote>
-                            {
-                                new()
-                                {
-                                    Subject = subject2
-                                }
-                            }
-                        }
-                    }
-                }
-            };
-
-            var releaseFootnotes = release.Footnotes.ToList();
-
-            var releaseSubjects = new List<ReleaseSubject>
-            {
-                new()
-                {
-                    Release = release,
-                    Subject = subject1
-                },
-                new()
-                {
-                    Release = release,
-                    Subject = subject2
-                }
-            };
+            var release = _releaseGenerator.Generate();
+            
+            var subject1Filters = _filterGenerator
+                .WithFilterGroups(_ => _filterGroupGenerator
+                    .WithFilterItems(_ => _filterItemGenerator.GenerateList(1))
+                    .GenerateList(1))
+                .GenerateList(3);
+            
+            var (subject1, subject2) = _subjectsGenerator
+                .ForIndex(0, s => s.SetFilters(subject1Filters))
+                .GenerateTuple2();
+
+            var subject1IndicatorGroup1 = _indicatorGroupGenerator
+                .WithIndicators(_indicatorGenerator.GenerateList(1))
+                .WithSubject(subject1)
+                .Generate();
+
+            var footnotes = _footnoteGenerator
+                .ForIndex(0, s => s
+                    .Set(f => f.Content, "Applies to subject 1")
+                    .SetSubjects(_ => ListOf(subject1)))
+                .ForIndex(1, s => s
+                    .Set(f => f.Content, "Applies to subject 1 filter 1")
+                    .SetFilters(_ => ListOf(subject1Filters[0])))
+                .ForIndex(2, s => s
+                    .Set(f => f.Content, "Applies to subject 1 filter 2 group 1")
+                    .SetFilterGroups(_ => ListOf(subject1Filters[1].FilterGroups[0])))
+                .ForIndex(3, s => s
+                    .Set(f => f.Content, "Applies to subject 1 filter 3 group 1 item 1")
+                    .SetFilterItems(_ => ListOf(subject1Filters[2].FilterGroups[0].FilterItems[0])))
+                .ForIndex(4, s => s
+                    .Set(f => f.Content, "Applies to subject 1 indicator 1")
+                    .SetIndicators(_ => ListOf(subject1IndicatorGroup1.Indicators[0])))
```

Comment:
In all of the cases where we're passing lambdas, we could just simplify these to:
```cs
SetFilters(ListOf(subject1Filters[0]))
// etc...
```

Same applies for lots of other cases in this test suite.

Reply:
Yup, assuming that we're happy keeping the various overloads of, for instance, `SetSubjects`, I can replace these with the more basic usages.

Reply:
Done ta